lock-api/src/main/java/com/palantir/lock/watch/ClientLockWatchEventLog.java

```diff
@@ -19,10 +19,12 @@
 import java.util.Optional;
 import java.util.Set;
 
-public interface LockWatchEventCache {
-    Optional<IdentifiedVersion> lastKnownVersion();
-    IdentifiedVersion processStartTransactionsUpdate(Set<Long> startTimestamps, LockWatchStateUpdate update);
-    void processUpdate(LockWatchStateUpdate update);
-    TransactionsLockWatchEvents getEventsForTransactions(Set<Long> startTimestamps,
-            Optional<IdentifiedVersion> version);
+import com.palantir.lock.LockDescriptor;
+
+public interface ClientLockWatchEventLog {
```

Comment:
This is a dodgy diff, as I moved the location of some of these files (`lock-api-objects` doesn't have tests, whereas `lock-api` does).